sqlstore/src/main/resources/db/migration/V8000__fix_box_size.sql

```diff
@@ -0,0 +1,28 @@
+-- Rename old table
+ALTER TABLE `BoxSize`
+    RENAME TO `BoxSize_old`;
+
+-- Recreate table
+CREATE TABLE `BoxSize` (
+  `boxSizeId` bigint(20) NOT NULL AUTO_INCREMENT,
+  `boxSizeRows` bigint(20) NOT NULL,
+  `boxSizeColumns` bigint(20) NOT NULL,
+  `scannable` boolean DEFAULT 0,
```

Comment:
Shouldn't be nullable (though I realize it was in the existing table)

Reply:
Fixed